docs/reference/eql/functions.asciidoc

```diff
@@ -856,6 +856,15 @@ Value to convert to a string. If `null`, the function returns `null`.
 +
 If using a field as the argument, this parameter does not support the
 <<text,`text`>> field data type.
++
```

Comment:
Actually, I think we need to restrict `string`s ability to handle complex boolean expressions altogether. I think the outcome of using this type of expression has a chance of throwing an exception.
@rw-access also gave a 👍 for imposing this restriction. Until this is added to EQL in the form of java code changes, I think your proposed documentation update here needs to wait a bit.

Reply:
That makes sense. Thanks @astefan.